engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ProcessCompilationError.scala

```diff
@@ -201,4 +201,15 @@ object ProcessCompilationError {
 
   case class CannotCreateObjectError(message: String, nodeId: String) extends ProcessCompilationError with InASingleNode
 
+  case class CustomParameterValidationError(message: String, description: String, paramName: String, nodeId: String) extends ParameterValidationError with InASingleNode
+
+  object CustomParameterValidationError{
+    def apply (message: String, description: String, paramName: String)(implicit nodeId: NodeId): CustomParameterValidationError = CustomParameterValidationError(message, description, paramName, nodeId.id)
+  }
+
+  case class CustomServiceValidationError(message: String, nodeId: String) extends ProcessCompilationError with InASingleNode
```

Comment:
Can we name it CustomNodeValidationError so that it can be reused for sources, sinks etc?

Reply:
Hmm, I've removed this one as well :) and reused CustomNodeError